data/json/items/book/misc.json

```diff
@@ -539,6 +539,23 @@
     "fun": 1,
     "flags": [ "INSPIRATIONAL" ]
   },
+  {
+    "id": "holybook_satanic",
+    "type": "BOOK",
+    "name": { "str": "The Satanic Bible", "str_pl": "copies of The Satanic Bible" },
+    "description": "A collection of essays, observations, and rituals published by Anton LaVey in 1969.",
+    "weight": "130 g",
+    "volume": "1 L",
```

Comment:
I'm absolutely willing to do that, but I was a little leery about breaking from the fact that every other holybook (including the one on Pastafairan) is exactly 1 L. It felt like a balance thing rather than a realism thing.

Reply:
I've seen many bibles that are less than a liter in size.  While there are certainly hardcover versions that are over a liter, I think that small paperbacks are probably far more common and that this would qualify as a bug with the other holy books.

Reply:
Yeah never assume something like weight or volume is balance based. If it doesn't make sense it's probably because the person implementing the JSON wasn't paying close attention the first time.